### DISCUSSION TITLE
How do I install MariaDB on M1 Mac?

### DISCUSSION THREAD

**Discussion by colette-exe** [Homebrew/discussions#3114] [Everyday usage]
Hello!

 I'm trying to follow the installation process on mariadb's website and whenever I get to "mysql_upgrade" I always receive an access denied error. What should I do to fix it? I also can't secure the installation because of that same error

![twitter_FO3aweIagAUIT1I](https://user-images.githubusercontent.com/64302033/160330921-7795a5e2-aa54-4c9f-b303-57845f8d5b6e.jpg)

This is the tutorial:

https://mariadb.com/resources/blog/installing-mariadb-10-1-16-on-mac-os-x-with-homebrew/

**Comment by SMillerDev** [comment]
You should specify the credentials the same way you would for the normal mysql command.
